src/Orc.FileAssociation/Services/FileAssociationService.cs

```diff
@@ -102,5 +105,15 @@ public async Task UndoAssociationFilesWithApplicationAsync(ApplicationInfo appli
             }
 
         }
+
+        public async Task OpenPropertiesWindowForExtensionAsync(string extension, IFileService fileService)
```

Comment:
Why passing IFileService to method intead resolving as dependency for FileAssociationService. Please, make appPath configurable instead

Reply:
make  virtual